.golangci.yml

```diff
@@ -0,0 +1,49 @@
+service:
+  golangci-lint-version: 1.21.0 # Keep this in sync with .circleci/config.yml and Jenkinsfile.pr
+
+run:
+  timeout: 5m
+  modules-download-mode: readonly
+
+linters-settings:
+  gofmt:
+    simplify: true
+  govet:
+    check-shadowing: true
+    enable-all: true
+
+linters:
+  disable-all: true
+  enable:
+    # - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - dupl
+    - errcheck
+    # - funlen
+    # - gochecknoinits
+    - goconst
+    - gocritic
+    - gocyclo
+    - gofmt
+    - goimports
+    - golint
+    # - gosec
```

Comment:
`gosec` complains about our `http.Get` calls using URLs provided by the GitHub client. No way around this as far as I can see.

Reply:
We can add a exclude rule for this in the code using `//nolint:gosec` (https://github.com/golangci/golangci-lint#nolint)

Reply:
Ugh... I'm all for extrinsic tests, but when a linter asserts something so wildly limiting that we have to start tuning it with comments eslint-style, I feel like we've lost the spirit of what's idiomatic Go.

Reply:
Hm, I see that comments are not the right way.

Maybe we just ignore this specific rule of `gosec` using
```yml
issues:
  exclude:
    - "G107: Potential HTTP request made with variable url"
```
?

Reply:
Sounds good!